lib/gush/worker.rb

```diff
@@ -51,7 +51,17 @@ def setup_job(workflow_id, job_id)
     def incoming_payloads
       payloads = {}
       job.incoming.each do |job_name|
-        payloads[job_name] = client.load_job(workflow.id, job_name).output_payload
+       job = client.load_job(workflow.id, job_name)
+       payloads[job.class.to_s] ||= []
+
+       if workflow.nameize_payloads
+         payload = {:id => job.name, :payload => job.output_payload}
```

Comment:
How about creating an helper method that will return the `output_payload` in a flat values array fomat? I know that is a one liner for every developer, but it makes it even more easier to develop a job with it (especially that I am using this :))